src/cuttlefish_rebar_plugin.erl

```diff
@@ -21,75 +18,120 @@
 %%
 %% -------------------------------------------------------------------
 
+%%
+%% @doc Rebar cuttlefish plugin to generate an application's default .conf
+%%      file as part of the build.
+%%
+%% For now, this is specific to Rebar2, but it's been refactored to allow it
+%% to be extended to work with Rebar3 as well.
+%% When we decide how we want to use it, we should do that.
+%%
 -module(cuttlefish_rebar_plugin).
 
 -export([
-    generate/2
+    generate/2,
+    init/1
```

Comment:
I think there is a seperate rebar3 cuttlefish plugin by @tsloughter (https://github.com/tsloughter/rebar3_cuttlefish) any reason for having a plugin in both here and a separate repo?